samples/net/lwm2m_client/src/lwm2m-client.c

```diff
@@ -57,10 +57,12 @@ static struct device *led_dev;
 static u32_t led_state;
 
 #if defined(CONFIG_NET_IPV6)
-static struct net_app_ctx udp6;
+static struct net_app_ctx app_udp6;
```

Comment:
Hmm, I seem to have talked about server side but the lwm2m is about client. So in client side this is not possible, sorry for confusion.